apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -61,23 +61,17 @@ export function ElectionScreen(): JSX.Element {
           </P>
         </div>
       </ElectionCard>
-      {isSystemAdministratorAuth(auth) && (
-        <UnconfigureMachineButton
-          isMachineConfigured
-          unconfigureMachine={unconfigureMachine}
-        />
-      )}
-      {isElectionManagerAuth(auth) && (
-        <React.Fragment>
-          <P>
-            Save the election package to the USB drive to configure VxSuite
-            components.
-          </P>
-          <P>
-            <ExportElectionPackageModalButton />
-          </P>
-        </React.Fragment>
-      )}
+      <div style={{ display: 'flex', gap: '1rem' }}>
```

Comment:
I think a 0.5rem gap might look better here (and IIRC that's what we've been doing elsewhere in VxAdmin)

Reply:
Ah right, the gap between the header buttons is 0.5rem, will update